apps/modernization-ui/src/components/Search/ExpandableResults.tsx

```diff
@@ -0,0 +1,41 @@
+import { NoData } from 'components/NoData';
+import {
+    PatientSearchResultAddress,
+    PatientSearchResultIdentification,
+    PatientSearchResultName
+} from 'generated/graphql/schema';
+import { useState } from 'react';
+import styles from './expandableResults.module.scss';
+
+type Props = {
+    results: PatientSearchResultAddress[] | PatientSearchResultIdentification[] | PatientSearchResultName[] | string[];
+    renderResult?: ({ _ }: any) => string;
```

Comment:
Issue: Use of `any`

Reply:
Talked to adam and working on refactoring the whole component so it is a feature of the DataTable by changing the row height when View more is clicked